pyiron_gui/project/project_browser.py

```diff
@@ -97,15 +98,370 @@ def __getattr__(self, item):
     def __repr__(self):
         return repr(self._wrapped_object)
 
-    def self_representation(self):
-        """Self representation of the wrapped object if known, else None is returned."""
-        if self._type is None:
-            return
-        if self._type == "structure":
-            return self._wrapped_object.plot3d()
-        if self._type == 'murnaghan':
+    @property
+    def gui(self):
+        return ObjectWidget(self).gui
+
+
+class AtomsWrapper(BaseWrapper):
+    def __init__(self, pyi_obj, project, rel_path=""):
+        super().__init__(pyi_obj, project, rel_path=rel_path)
+        self._name = 'structure'
+
+    @property
+    def gui(self):
+        return AtomsWidget(self).gui
+
+
+class MurnaghanWrapper(BaseWrapper):
+    def __init__(self, pyi_obj, project, rel_path=""):
+        super().__init__(pyi_obj, project, rel_path=rel_path)
+        self._name = 'murnaghan'
+
+    @property
+    def gui(self):
+        return MurnaghanWidget(self).gui
+
+
+class ObjectWidget:
+    def __init__(self, obj):
+        self._obj = obj
+        self._output = widgets.Output()
+        self._box = widgets.VBox()
+
+    def refresh(self):
+        self._output.clear_output()
+        with self._output:
+            print(" ")
```

Comment:
Not objecting, but what's this do?

Reply:
The `widget.Output().clear_output()` sometimes does not refresh what is actually shown in the output widget. Therefore, printing an empty string ensures that the previous output disappears.